rust/src/ostree_utils.rs

```diff
@@ -1,9 +1,33 @@
 //! Utility helpers or workarounds for incorrectly bound things in ostree-rs
 // SPDX-License-Identifier: Apache-2.0 OR MIT
 
+use glib;
 use glib::translate::*;
 use std::ptr;
 
+pub(crate) fn repo_resolve_ref_optional(
```

Comment:
Probably worth mentioning in a comment/docstring here too we can drop this once that OSTree PR can be used?

Reply:
This whole file is that.